docs/faq.md

```diff
@@ -332,7 +332,7 @@ In Maven, you can use the `maven-resources-plugin` to copy files to your extra d
   ...
   <plugin>
     <artifact>maven-resources-plugin</artifact>
-    <version>3.1.1</version>
+    <version>3.2.0</version>
```

Comment:
I realized there doesn't exist 3.1.1 for `maven-resources-plugin`. I think it was a wrong update during the post release process in the past.

Reply:
That is a great catch!